react-app/tailwind.config.js

```diff
@@ -9,32 +9,30 @@ module.exports = {
         "2lg": "32.5rem",
       },
       colors: {
-        likecoin: {
-          green: "#28646E",
-          black: "#222222",
-          darkgrey: "#666666",
-          lightgreen: "#A5B6B9",
-          darkgreen: "#1A4951",
-          grey: "#D8D8D8",
-          gold: "#F0ECDA",
-          darkgrey: "#666666",
-          lightgrey: "#E8E8E8",
-          lightergrey: "#F8F8F8",
-          secondarygreen: "#E0EFEB",
-          "primary-bg": "#F8F7F4",
-          black: "#222222",
-          gradient: {
-            from: "#D7EFE7",
-            to: "#EFE7B8",
-          },
-          "vote-color": {
-            yes: "#6DCDBC",
-            no: "#C72F2F",
-            veto: "#24272B",
-            abstain: "#B7B7B7",
-          },
-          yellow: "#FFD951",
+        green: "#28646E",
```

Comment:
this overlaps tailwind's default color names? maybe add keep the prefix but use something neutral like `theme` or `app`?